### ISSUE TITLE
Bug: `<text>` component throws type errors

### ISSUE THREAD

**Issue by trezy** [pixijs/pixi-react#500]
### Current Behavior

The `<text>` component throws a type error when using the `text` property.

### Expected Behavior

The `<text>` component **does not** throw a type error when using the `text` property.

### Steps to Reproduce

```jsx
<text text={'text'} />
```

### Environment

- **`@pixi/react` version**: *8.0.0-dev.f3abf87*
- **`pixi.js` version**: *8.2.1*
- **`React` version**: *18.3.1*
- **`ReactDOM` version**: *18.3.1*

### Additional Information

I'm pretty confident the issue is the order of overloads on the `Text` constructor in Pixi.js. When we're dynamically building the types for Pixi React, we can't choose which overload to use. I think Typescript is just picking the last overload (which is deprecated) by default. This overload doesn't accept `TextOptions` as the first property like most other classes, so it doesn't expose the appropriate props on the `<text>` copmponent.

https://github.com/pixijs/pixijs/blob/dev/src/scene/text/Text.ts#L40-L46

**Comment by trezy** [comment]
Fixed in `8.0.0-dev.6b268b5`.

**Comment by utenma** [comment]
On v9.beta13 react treats `text` tags as `JSX.IntrinsicElements.text: React.SVGTextElementAttributes<SVGTextElement>`
![image](https://github.com/user-attachments/assets/9d543b29-3c31-4a93-b217-d4168836c2ef)

tried the following but cannot resolve PixiReactElementProps
```tsx
declare namespace JSX {
    interface IntrinsicElements {
        ['text']: PixiReactElementProps<typeof Text>
    }
}
```
